### PR TITLE
Add python script to generate previous releases table

### LINKED ISSUE #369: Automate Previous Releases Table
At http://fuelcycle.org/previous/index.html we have a table of zips/tarballs from previous releases.  It would be nice to automate this somehow so it does not have to be manually updated after each release

### PR DESCRIPTION
**Pull request by bennibbelink**
This will require a rebase after #381 is merged (for the `mamba update --all` line).  Runs a python script to construct the previous releases table so we don't have to manually update it on release.  Closes #369

### CODE REVIEW

**Comment by bennibbelink** [comment]
The table is in `/previous/index.html` for reference

File: source/releases.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+     - Tar'''
+    for release in releases:
+        version = release['tag_name']
+        date = release['published_at']
+        date_object = datetime.strptime(date, "%Y-%m-%dT%H:%M:%SZ")
+        date_formatted = date_object.strftime("%m/%d/%Y")
+        zip_url = release['zipball_url']
+        tar_url = release['tarball_url']
+        table_text += f'''
+   * - {version}
```
Comment (gonuke) [inline source/releases.py]: How about we add a link to the Github Release page?
```suggestion
        tar_url = release['tarball_url']
        web_url = release['html_url']
        table_text += f'''
   * - `{version} <{web_url}>`
```

**Review by gonuke — CHANGES_REQUESTED** [review]
Thanks - I added a suggestion for a  link to the Github release page as well...

File: source/releases.py (hunk 1 above)
Comment (bennibbelink) [reply source/releases.py]: Good idea, LGTM

**Review by gonuke — APPROVED** [review]
Thanks @bennibbelink

**Pull request merged**